Mathlib/LinearAlgebra/BilinearMap.lean

```diff
@@ -319,6 +320,13 @@ theorem llcomp_apply (f : Nₗ →ₗ[R] Pₗ) (g : M →ₗ[R] Nₗ) (x : M) :
 theorem llcomp_apply' (f : Nₗ →ₗ[R] Pₗ) (g : M →ₗ[R] Nₗ) : llcomp R M Nₗ Pₗ f g = f ∘ₗ g := rfl
 #align linear_map.llcomp_apply' LinearMap.llcomp_apply'
 
+lemma inj_llcomp_flip_apply_of_surj (f : M →ₗ[R] Nₗ) (h : Surjective f) :
+    Injective <| (LinearMap.llcomp R M Nₗ Pₗ).flip f := by
+  intro g₁ g₂ hg
```

Comment:
Arguably this should just be stated as injectivity of `. ∘ₗ g`

Reply:
Indeed I slightly prefer this; now changed.